airbyte-ci/connectors/connectors_qa/poetry.lock

```diff
@@ -116,13 +117,13 @@ files = [
 
 [[package]]
 name = "beartype"
-version = "0.17.2"
+version = "0.17.1"
```

Comment:
This is specifically the one that was failing - it's a lockfile only change because it comes from the base image dependency.